saspy/sas_magic.py

```diff
@@ -54,22 +54,40 @@ def SAS(self, line, cell):
             run;
         """
         
+        mva = self.mva
+        if len(line) and line in self.shell.user_ns:  # session supplied
+            _mva = self.shell.user_ns[line]
+            if isinstance(_mva, SASsession):
+                mva = _mva
+            else:
+                return 'Invalid SAS Session object supplied'
+        elif len(line) and not line in self.shell.user_ns:  # string supplied but not a session
+            return 'Invalid SAS Session object supplied'
+        else:  # no string should default to last session called
```

Comment:
The comment should read
```
# no string should default to unnamed session
```
This is comment from an earlier iteration where I was trying to use the last session when you didn't provide a name but it proved too confusing.
Everything else looks good.

Reply:
Thanks - good eye, I neglected to double check the inherited comments there.